Assets/MixedRealityToolkit/Utilities/Lines/Renderers/MixedRealityLineRenderer.cs

```diff
@@ -46,6 +46,16 @@ public bool RoundedCaps
         [HideInInspector]
         private LineRenderer lineRenderer = null;
 
+        [Header("Texture Tiling")]
+        [SerializeField]
+        [Tooltip("Tiles the material on the line renderer by world length. Use if you want the texture size to remain constant regardless of a line's length.")]
+        private bool tileMaterialByWorldLength = false;
```

Comment:
Is this property specific to the material that's being used? In other words, does it assume that a specific material is being used on the Line renderer? I'm noticing the following line:

```
tilingPropertyBlock.SetVector("_MainTex_ST", tilingPropertyVector);
```

If it is a behavior specific to the Shell line tether, I wonder if it would make sense to instead extend the class (ShellTetherLineRenderer or something)

Reply:
That's a great question. This is the default shader property name Unity (and the MRTK) uses for texture tiling and offset of the main texture (albedo). So, it should be compatible with most shaders a developer choses. But, we could always expose that property name if desired?